database/seeds/learn.php

```diff
@@ -77,6 +77,21 @@
                     'url' => 'https://mastery.games/p/flexbox-zombies',
                     'type' => 'game',
                 ],
+                [
+                    'name' => 'Frontend Masters Web Development Bootcamp',
+                    'url' => 'https://frontendmasters.com/bootcamp/',
+                    'type' => 'course',
+                ],
+                [
```

Comment:
I think CSS Grid and Flexbox are both "bonus." Do we have the idea of bonus represented in these seeds yet?

Reply:
OK, I just added the ability to mark them as bonus. Just add the "bonus" keys and set them to true.

Reply:
It seems to be a nice addition.